components/scream/src/physics/shoc/shoc_ic_cases.cpp

```diff
@@ -202,8 +202,8 @@ FortranData::Ptr make_standard(const Int shcol, Int nlev, Int num_qtracers) {
     // Compute pressure differences and host_dse / exner.
```

Comment:
Minor:  should the line change to "host_dse * inv_exner"?

Reply:
Yes it should.